src/frontend/components/Felleskomponenter/Barnehagemodal/BarnehageplassPeriodeModal.tsx

```diff
@@ -178,12 +184,12 @@ export const BarnehageplassPeriodeModal: React.FC<Props> = ({
                             label={
                                 <TekstBlock block={barnehageplassTekster.antallTimer.sporsmal} />
                             }
-                            bredde={'S'}
+                            htmlSize={15}
                         />
 
-                        <Alert variant={'info'} inline>
+                        <StyledAlert variant={'info'} inline>
```

Comment:
Kunne man byttet ut `<SkjemaGruppe>` med f.eks komponentgruppe for å slippe denne stylingen? Da får man også kvitt enda import fra gammelt design system

Reply:
Kan man også fjerne denne tenkte jeg på: 
`const StyledAlert = styled(Alert)`
    margin-top: 1rem;
`;`
? (at man kanskje ikke trenger margin-top dersom man bruker `<KomponentGruppe`, men er ikke sikker